chcemvediet/apps/obligees/tests/test_forms.py

```diff
@@ -115,102 +121,106 @@ def test_submitted_with_nonexisting_obligee_name(self):
                 <input class="autocomplete" data-autocomplete-url="{url}" id="id_obligee" name="obligee" type="text" value="invalid">
                 """.format(url=reverse(u'obligees:autocomplete')), rendered)
 
-    @unittest.skip(u'FIXME')
-    def test_to_python_is_cached(self):
-        names = [u'aaa', u'bbb', u'ccc', u'ddd']
-        oblgs = [self._create_obligee(name=n) for n in names]
-        field = ObligeeField()
-
-        # Valid value
-        with self.assertNumQueries(1):
-            self.assertEqual(field.clean(u'bbb'), oblgs[1])
-        with self.assertNumQueries(0):
-            self.assertEqual(field.clean(u'bbb'), oblgs[1])
-
-        # Invalid value
-        with self.assertNumQueries(1):
-            with self.assertRaises(ValidationError):
-                field.clean(u'invalid')
-        with self.assertNumQueries(0):
-            with self.assertRaises(ValidationError):
-                field.clean(u'invalid')
-
 class ObligeeFieldWithObligeeWidgetWidget(ObligeesTestCaseMixin, TestCase):
     u"""
     Tests ``ObligeeField`` with ``ObligeeWidget`` widget.
     """
 
     class Form(forms.Form):
-        obligee = ObligeeField(
-                widget=ObligeeWidget(),
-                )
+        obligee = ObligeeField()
 
     class FormWithWidgetAttrs(forms.Form):
         obligee = ObligeeField(
-                widget=ObligeeWidget(attrs={
+                widget=ObligeeWidget(input_attrs={
                     u'class': u'custom-class',
                     u'custom-attribute': u'value',
                     }),
                 )
 
 
-    @unittest.skip(u'FIXME')
     def test_new_form(self):
         form = self.Form()
         rendered = self._render(u'{{ form }}', form=form)
+        lorem = squeeze(render_to_string(u'obligees/texts/widget_no_email.txt'))
         self.assertInHTML(u'<label for="id_obligee">Obligee:</label>', rendered)
         self.assertInHTML(u"""
-                <input class="autocomplete" data-autocomplete-url="{url}" id="id_obligee" name="obligee" type="text">
```

Comment:
`id="id_obligee"` sa vyrendruje do kmenoveho attributut `<div id="id_obligee" class="chv-obligee-widget" >` . Kvoli sposobu ako kontroluje `assertInHtml` by bolo potrebne pridat aj vsetko co je vo vnutry div. Mam teda pridat aj kontrolu `id="id_obligee"` alebo to necham tak ako som to spravil teraz?